src/main/java/uk/gov/ch/oauth/Oauth2.java

```diff
@@ -227,6 +220,7 @@ public void invalidateSession(Session chSession) {
         }
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
Not saying this is wrong, just curious, what is the warning here?

Reply:
casting the Object into a Map<String,Object>. It could throw errors if the Object isn't actually a Map.